### ISSUE TITLE
Segmentation fault when creating projects with `npm create hono@latest`

### ISSUE THREAD

**Issue by prosenjitjoy** [honojs/create-hono#35]
## Context
Running a simple command like `npm create hono@latest ./my-app -- --template cloudflare-pages` gives
```
create-hono version 0.7.0
✔ Using target directory … ./my-app
Segmentation fault (core dumped)
```

## Expected
Should work as intended to create projects using different templates.

## Device
Kernel: Linux 6.8.7
Distro: Fedora 40
Node: 20.12.2
NPM: 10.5.0
Git: 2.44.0

**Comment by yusukebe** [comment]
Hi @prosenjitjoy 

Is `git` installed in your machine?

**Comment by prosenjitjoy** [comment]
> Hi @prosenjitjoy
> 
> Is `git` installed in your machine?

はい、先生

**Comment by ryuapp** [comment]
Probably related to the following: https://github.com/sindresorhus/ora/issues/229

**Comment by yusukebe** [comment]
Hi @prosenjitjoy @ryuapp 

It seems to be related to that `ora` issue. The easiest way to resolve this issue is to not use `ora`, but there is no good spinner library, and I'd like to keep using `ora`.

@ryuapp Or do you know any other library or idea to solve this?

**Comment by ryuapp** [comment]
Hi @yusukebe 
I found `nanospinner`(https://github.com/usmanyunusov/nanospinner).
I tried it on local and it looks fine.

I can't confirm whether this solves segmentation fault problems because I can't reproduce it, but I think it will work.

**Comment by yusukebe** [comment]
@ryuapp 

That's good, only on dependency! If you can, could you create the PR to replace `ora` with `nanospinner`?

**Comment by yusukebe** [comment]
Hi @prosenjitjoy 

I've released `v0.7.1` now, which includes the fix that replaces `ora` with `nanospinner` by @ryuapp #37 

Could you try it and let us know the result?

**Comment by prosenjitjoy** [comment]
It is now working. Thanks.
